src/main/java/org/terasology/dialogs/DialogSystem.java

```diff
@@ -152,9 +153,11 @@ public void showDialog(ShowDialogEvent event, EntityRef character) {
         }
 
         window.setDocument(documentData);
+        String imageURN = templateEngine.transform(r.responseImage);
+        TextureRegion image = null;//= (imageURN.equals(null) ? "" : imageURN);
```

Comment:
@skaldarnar what I want to do here is to set the default image TextureRegion if the user hasn't given an image themself, while when the user has given an image then to set that (String)image as the TextureRegion. My issue is that I can't seem to find (through my research to other files) how a TextureRegion is set, or how you invert a String to TextureRegion.